src/Server/Repositories/ClaraJobRepository.cs

```diff
@@ -0,0 +1,236 @@
+                 })
+                 .ConfigureAwait(false);
+        }
+
+        private InferenceJob CreateInferenceJob(Job job, string jobName, IList<InstanceStorageInfo> instances)
+        {
+            Guard.Against.Null(job, nameof(job));
+            Guard.Against.Null(jobName, nameof(jobName));
+            Guard.Against.Null(instances, nameof(instances));
+
+            var targetStoragePath = string.Empty; ;
+            if (_fileSystem.Directory.TryGenerateDirectory(_fileSystem.Path.Combine(_configuration.Value.Storage.Temporary, "jobs", $"{job.JobId}"), out targetStoragePath))
+            {
+                _logger.Log(LogLevel.Information, $"Job payloads directory set to {targetStoragePath}");
+                return new InferenceJob(targetStoragePath, job) { Instances = instances };
+            }
+            else
+            {
+                throw new JobStoreException($"Failed to generate a temporary storage location");
+            }
+        }
+
+        private void MakeACopyOfPayload(InferenceJob request)
+        {
+            _logger.Log(LogLevel.Information, $"Copying {request.Instances.Count} instances to {request.JobPayloadsStoragePath}.");
+            var files = new Stack<InstanceStorageInfo>(request.Instances);
+            var retrySleepMs = 1000;
+            var retryCount = 0;
+
+            while (files.Count > 0)
+            {
+                try
+                {
+                    var file = files.Peek();
+                    var destPath = _fileSystem.Path.Combine(request.JobPayloadsStoragePath, $"{file.SopInstanceUid}.dcm");
+                    _fileSystem.File.Copy(file.InstanceStorageFullPath, destPath, true);
+                    _logger.Log(LogLevel.Debug, $"Instance {file.SopInstanceUid} moved to {destPath}");
+                    files.Pop();
+                }
+                catch (IOException ex) when ((ex.HResult & 0xFFFF) == 0x27 || (ex.HResult & 0xFFFF) == 0x70)
+                {
+                    if (++retryCount > 3)
+                    {
+                        _logger.Log(LogLevel.Error, ex, $"Error copying file to {request.JobPayloadsStoragePath}; destination may be out of disk space.  Exceeded maximum retries.");
+                        throw;
+                    }
+                    _logger.Log(LogLevel.Error, ex, $"Error copying file to {request.JobPayloadsStoragePath}; destination may be out of disk space, will retry in {retrySleepMs}ms");
+                    Thread.Sleep(retryCount * retrySleepMs);
+                }
+                catch (Exception ex)
+                {
+                    _logger.Log(LogLevel.Error, ex, $"Failed to copy file {request.JobPayloadsStoragePath}");
+                    throw;
+                }
+            }
+
+            _logger.Log(
+                files.Count == 0 ? LogLevel.Information : LogLevel.Warning,
+                $"Copied {request.Instances.Count - files.Count} files to {request.JobPayloadsStoragePath}.");
```

Comment:
`$"Copied {request.Instances.Count - files.Count} files to {request.JobPayloadsStoragePath}."` => `$"Copied {request.Instances.Count - files.Count:D} files to '{request.JobPayloadsStoragePath}'."`

Reply:
Fixed 🙏